README.md

```diff
@@ -1 +1,24 @@
-# anitab-org.github.io
\ No newline at end of file
+# anitab-org.github.io
+
+This is the main website for **AnitaB.org Open Source**. It contains information about our various open source projects, the programs we participate in, information about how to contribute, and how to contact us.This website acts as an interface to help introduce people to the organization's Open Source community, and make the transition from newcomers to contributors easier.
+
+
+## Contributing
+
+**This project is under active development**
+
+Please read our [Code of Conduct](https://github.com/anitab-org/anitab-org.github.io/blob/develop/code_of_conduct.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-org.github.io/blob/develop/reporting_guidelines.md) thoroughly.
+
+## Branches
+
+* **master** This contains the code which will be deployed into production.
```

Comment:
This contains a deployment of web along with `gh-pages` branch, `production` is a branch that triggers deployment.

Reply:
Made the necessary changes, pls do take a look. :sweat_smile:

Reply:
* **master** and **gh-pages** These branches contains the deployment of the website. 
* **develop** This contains the latest code. All the contributing PRs must be sent to this branch.
* **production** Merge develop into production triggers deployment of  the website.

Reply:
done 👍